apps/ios/Shared/Views/ChatList/ServersSummaryView.swift

```diff
@@ -174,11 +174,11 @@ struct ServersSummaryView: View {
 
     private func smpSubsSection(_ totals: SMPTotals) -> some View {
         Section {
-            infoRow("Connections subscribed", numOrDash(totals.subs.ssActive))
+            infoRow("Active connections", numOrDash(totals.subs.ssActive))
             infoRow("Total", numOrDash(totals.subs.total))
         } header: {
             HStack {
-                Text("Message subscriptions")
+                Text("Message push")
```

Comment:
I think "push" is more associated with sent messages - it's not clear that you mean "push from server" here. Maybe "Message reception"
```suggestion
                Text("Message reception")
```